Mage/src/main/java/mage/Mana.java

```diff
@@ -464,7 +552,11 @@ public int count() {
      * @return the total count of all colored mana.
      */
     public int countColored() {
-        return red + green + blue + white + black + any;
+        return white.getAmount()
+                + blue.getAmount()
+                + black.getAmount()
+                + red.getAmount()
+                + green.getAmount();
```

Comment:
Missing any?

Reply:
I really hope that's the biggest mistake I made